initial_setup/__init__.py

```diff
@@ -113,11 +121,26 @@ def __init__(self, gui_mode):
         setup_ifcfg_log()
 
     @property
+    def external_reconfig(self):
```

Comment:
But I don't see the point of making this @property and having it just mirror the self._external_reconfig, it isn't actually doing anything more than a plain old attribute would.